src/components/OrderableList/OrderableList.stories.tsx

```diff
@@ -0,0 +1,138 @@
+            },
+            {
+                id: "2",
+                type: "item",
+                content: (
+                    <>
+                        <Checkbox value={CheckboxState.Checked} label="List Item With Checkbox" />
+                    </>
+                ),
+                alt: "Checkbox Item",
+            },
+            {
+                id: "3",
+                type: "item",
+                content: <Button>List Item With Button</Button>,
+                alt: "List Item With Checkbo",
+            },
+            {
+                id: "4",
+                type: "item",
+                content: (
+                    <div>
+                        <p>List Item With multiple focusable elements</p>
+                        <div className="tw-flex tw-justify-around tw-mt-2">
+                            <Checkbox value={CheckboxState.Unchecked} label="First Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Second Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Third Input" />
+                            <Checkbox value={CheckboxState.Unchecked} label="Fourth Input" />
+                        </div>
+                    </div>
+                ),
+                alt: "text",
+            },
+            { id: "5", type: "item", content: "Five", alt: "Five" },
+        ],
+    });
+
+    const onMove = (selectedGridItemKeys: React.Key[], gridItemLocation: ItemDropTarget) => {
+        if (gridItemLocation.dropPosition === "before") {
+            list.moveBefore(gridItemLocation.key, selectedGridItemKeys);
+        } else {
+            list.moveAfter(gridItemLocation.key, selectedGridItemKeys);
+        }
+    };
+
+    return (
+        <div style={{ width: "600px" }} className="tw-m-auto">
+            <OrderableList
+                items={list.items}
+                onMove={onMove}
+                showFocusRing
+                dragDisabled={dragDisabled}
+                renderContent={({ value, prevKey, nextKey }, { componentDragState, isFocusVisible }) => {
+                    return (
+                        <div
+                            className={merge([
+                                "tw-break-word tw-border tw-border-solid tw-rounded tw-p-3",
+                                componentDragState === ItemDragState.Dragging &&
+                                    "tw-bg-black-10 tw-border-black-20 tw-opacity-75",
+                                componentDragState === ItemDragState.Preview &&
+                                    "tw-bg-white tw-border-violet-70 tw-border-4",
+                                componentDragState === ItemDragState.Idle && "tw-border-black-20",
+                                isFocusVisible && "tw-bg-violet-90",
+                                (nextKey === null || prevKey === null) && "tw-bg-green-20",
+                            ])}
+                        >
```

Comment:
Could you move this to its own function?

Reply:
done